### ISSUE TITLE
which model to download?

### ISSUE THREAD

**Issue by JaeDukSeo** [ecnuljzhang/brush-your-text#7]
https://huggingface.co/runwayml/stable-diffusion-v1-5/tree/main

**Comment by JaeDukSeo** [comment]
Do we have to replicate the same env? So exact same number of folders and download everything

**Comment by Lazy5233** [comment]
Yes, you should download all the files via huggingface's official download tool, huggingface-cli, and then it will work fine!

**Comment by woshing700** [comment]
![image](https://github.com/user-attachments/assets/78334b7a-a412-4983-9e2e-7fc089761a79)


原本https://huggingface.co/runwayml/stable-diffusion-v1-5/tree/main    现在打不开了。
我进入了https://huggingface.co/stable-diffusion-v1-5/stable-diffusion-v1-5
发现是原本的存储库已经弃用。那用现在的https://huggingface.co/stable-diffusion-v1-5/stable-diffusion-v1-5
这个模型可以用吗？
